BloodDonor/Modules/BloodCenter/BloodCenter.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  BloodCenter.swift
+//  BloodDonor
+//
+//  Created by Adam Jurkiewicz on 31/05/2023.
+//
+
+import Foundation
+import Alamofire
+import MapKit
+
+struct BloodCenterBankDetails: Decodable {
+    var bloodType: String
+    var capacity: String
+    var sourceDatetime: String
+}
+
+struct BloodCenters: Decodable, Identifiable {
+    var id: Int
+    var name: String
+    var streetName: String
+    var streetNumber: String
+    var postalCode: String
+    var city: String
+    var voivodeship: String
+    var geoCoordinates: String
+    var phoneNumber: String
+    var openFrom: String?
+    var openTo: String?
+    var bloodCenterDetails: [BloodCenterBankDetails]?
+
+}
+
+class BloodCenterDataService {
+    var apiBaseUrl =  "https://api.pre.gwn.adamjurkiewicz.com"
+    let apiToken: String
+    let headers: HTTPHeaders
+    
+    init() {
+        apiToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJibG9vZGRvbm9yIiwiY29udGV4dCI6eyJ1c2VyIjp7InVzZXJJZCI6MSwiZGlzcGxheU5hbWUiOiJhZGFtaiJ9fSwiaWF0IjoxNjg1NTUwNjI1LCJleHAiOjE2ODYxNTU0MjV9.eauTNijJm2fXBjvKZFY0A3tqdLTUwKnk0jcGqY7aqEg"
```

Comment:
This is temporary solution, yes?

Reply:
Yep - this allowed me to simulate how would Swift map the JSON that it receives from the API (and it did well in that regard!).